bids-validator/bids_validator/rules/file_level_rules.json

```diff
@@ -557,9 +543,9 @@
   },
 
   "pet_blood_data": {
-    "regexp": "^\\/(sub-[a-zA-Z0-9]+)\\/(?:(ses-[a-zA-Z0-9]+)\\/)pet\\/(sub-[a-zA-Z0-9]+)_(?:(ses-[a-zA-Z0-9]+))|((@@@_pet_ext_@@@))$",
+    "regexp": "^[\\/\\\\](sub-[a-zA-Z0-9]+)[\\/\\\\](?:(ses-[a-zA-Z0-9]+)[\\/\\\\])pet[\\/\\\\](sub-[a-zA-Z0-9]+)(?:(_ses-[a-zA-Z0-9]+))?(?:_task-[a-zA-Z0-9]+)?(?:_trc-[a-zA-Z0-9]+)?(?:_rec-[a-zA-Z0-9]+)?(?:_run-[0-9]+)?(?:_recording-[a-zA-Z0-9]+)?(@@@_pet_ext_@@@)$",
     "tokens": {
-      "@@@_pet_ext_@@@": ["_blood\\.tsv"]
+      "@@@_pet_ext_@@@": ["_blood\\.json", "_blood\\.tsv"]
```

Comment:
Very nice.

Reply:
I think we should do the same thing to the "pet" rule, removing "pet_data" and updating the "pet" regexp to `^[\\/\\\\](sub-[a-zA-Z0-9]+)[\\/\\\\](?:(ses-[a-zA-Z0-9]+)[\\/\\\\])pet[\\/\\\\](sub-[a-zA-Z0-9]+)_(?:(ses-[a-zA-Z0-9]+))?(?:_task-[a-zA-Z0-9]+)?(?:_trc-[a-zA-Z0-9]+)?(?:_rec-[a-zA-Z0-9]+)?(?:_run-[0-9]+)?(?:_recording-[a-zA-Z0-9]+)?(@@@_pet_ext_@@@)$`